openkfw/TruBudget
# Add tags to projects

# Changes
A new feature has been added which makes it now possible to add tags to projects. This feature consists of 3 parts: 
- API: Add tags as new attribute to projects
- UI: Add, edit and display tags for projects
- E2E-test: Create new tests for this feature

Closes #343 
Closes #342 

# How to test
Create or edit a project and add, remove tags - they should be displayed on the overview table and the project details page
